xyz.safeworlds.hiit.yml

```diff
@@ -0,0 +1,40 @@
+id: xyz.safeworlds.hiit
+runtime: org.gnome.Platform
+runtime-version: "45"
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable # 23.08
+- org.freedesktop.Sdk.Extension.llvm16
+command: hiit
+finish-args:
+- --share=ipc
+- --socket=fallback-x11
+- --socket=wayland
+- --socket=pulseaudio
+- --device=dri
+- --env=RUST_LOG=gtk_rust_template=debug
+- --env=G_MESSAGES_DEBUG=none
+- --env=RUST_BACKTRACE=1
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/llvm16/bin
+  prepend-ld-library-path: /usr/lib/sdk/llvm16/lib
+  build-args:
+  - --share=network
```

Comment:
this is not allowed and the build will fail because of it.

Reply:
Fixed it via cargo-sources.json